napari_aicsimageio/napari.yaml

```diff
@@ -0,0 +1,44 @@
+name: napari-aicsimageio
+schema_version: 0.1.0
+contributions:
+  commands:
+  - id: napari-aicsimageio.get_reader
+    title: Get AICSImageIO Reader
+    python_name: napari_aicsimageio.core:get_reader
+  readers:
+  - command: napari-aicsimageio.get_reader
+    filename_patterns: [
```

Comment:
This is rough but understandable. Should become easier to generate in aicsimageio v5 though because plugins report which filetypes they support.

Reply:
it's not strictly necessary... we could also use `filename_patterns: ["*"]` ... but this is preferable, as it will allow napari to do things like this make suggestions to a user that _doesn't_ have aicsimageio installed that "there's a plugin that can help with this"